README.md

```diff
@@ -160,7 +160,10 @@ An API is made up of multiple resources. Each resource has a list of endpoints t
 
 - **name**: A `string` containing the key or name of this parameter that will be sent in the request body or injected into the URI.
 - **required**: A `bool` (`true` or `false`) which flags this parameter as required.
-- **type**: A `string` containing one of: `string`, `integer`, `boolean`, `resumable-upload`, `hash`, `array[hash]`, `array[string]`, `array[boolean]`, `array[integer]`, `stripe_token`, or `enum`. If `enum`, specify `enumValues` property on param containing an array of strings to display as options to the user.
+- **type**: A `string` containing one of: `string`, `integer`, `boolean`, `resumable-upload`, `hash`, `array[hash]`, `array[string]`, `array[boolean]`, `array[integer]`, `stripe_token`, `custom-object`, or `enum`.
+  - `hash` and `array[hash]` will require a child set of `params`
```

Comment:
The idea of `hash` and `object` are one and the same. I think it should be `object`, `array[object]`, and `custom-object`.

Aside: I'm not sure why we called it a hash in the first place. "Hash table" is a behind-the-scenes implementation detail about how you can structure a `map`/`dictionary`/`associative array`/(JavaScript object).